notebooks/egov/complaint_openai_assistant.py

```diff
@@ -0,0 +1,161 @@
+import openai
+from openai._client import OpenAI
+client = OpenAI()
+
```

Comment:
What is this file ?

Reply:
these all were initial files egov people shared with us. I'm removed the entire directory now. There's no use for it for us now